src/tests/suite/daos_test_common.c

```diff
@@ -903,12 +914,11 @@ daos_kill_server(test_arg_t *arg, const uuid_t pool_uuid,
 		       arg->srv_disabled_ntgts - 1, svc->rl_nr);
 
 	/* build and invoke dmg cmd to stop the server */
-	if (arg->dmg_config == NULL)
-		dts_create_config(dmg_cmd, "dmg system stop -i --ranks=%d "
-				  "--force", rank);
-	else
-		dts_create_config(dmg_cmd, "dmg system stop -i --ranks=%d "
-				  "--force -o %s", rank, arg->dmg_config);
+	if (arg->dmg_config != NULL)
+		dts_create_config(dmg_options, " -o %s", arg->dmg_config);
+
```

Comment:
what is the purpose of dmg_options buffer?   Why not just add -o %s to dmg_cmd  ?

Reply:
Not sure dts_create_config will clear dmg_cmd every time called, or will append the new content.  
        va_start(ap, format);
        count = vsnprintf(buf, DTS_CFG_MAX, format, ap);
        va_end(ap);

Reply:
could create dts_append_config for this purpose?   Seems like a reasonable use case and would avoid duplication

Reply:
Will be helpful to have a dts_append_config. Thought about and looked around for that :-(. Let me work on it hoping it will benefit other user cases.

Reply:
I was suggesting adding it right next to the definition for dmg_create_config.